docs/source/components/nodes/color_camera.rst

```diff
@@ -0,0 +1,83 @@
+ColorCamera
+===========
+
+ColorCamera node is a source of :ref:`image frames <ImgFrame>`. You can control in at runtime with the :code:`InputControl` and :code:`InputConfig`.
+
+How to place it
+###############
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    manip = pipeline.createColorCamera()
+
+  .. code-tab:: c++
+
+    dai::Pipeline pipeline;
+    auto imageManip = pipeline.create<dai::node::ColorCamera>();
+
+
+Inputs and Outputs
+##################
+
+.. code-block::
+
+  InitialControl ┌───────────────────┐   Still
+  ──────────────►│                   ├───────────►
+                 │                   │
+  InputConfig    │                   │   Preview
+  ──────────────►│    ColorCamera    ├───────────►
+                 │                   │
+  InputControl   │                   │   Video
+  ──────────────►│                   ├───────────►
```

Comment:
Use same names as in API - do not capitalize
`inputControl`, `inputConfig`, `video`, `preview`, `still`
And remove `InitialControl` - its not an input.